models/tokens/bnb/tokens_bnb_bep20.sql

```diff
@@ -270,7 +270,7 @@ SELECT LOWER(contract_address) AS contract_address, symbol, decimals
 ,('0xb6c53431608e626ac81a9776ac3e999c5556717c', 'TLOS', 18)
 ,('0xeca41281c24451168a37211f0bc2b8645af45092', 'TPT', 4)
 ,('0x7af173f350d916358af3e218bdf2178494beb748', 'TRADE', 18)
-,('0x85eac5ac2f758618dfa09bdbe0cf174e7d574d5b', 'TRX', 18)
+,('0xCE7de646e7208a4Ef112cb6ed5038FA6cC6b12e3', 'TRX', 18)
```

Comment:
lowercase
```suggestion
,('0xce7de646e7208a4ef112cb6ed5038fa6cc6b12e3', 'TRX', 18)
```

Reply:
I changed the address on the tokens table since it appears 'TRX' has a new address. Would if break anything?